scapy/sendrecv.py

```diff
@@ -44,12 +47,41 @@ class debug:
 ####################
 
 
+def _sndrcv_snd(pks, timeout, inter, verbose, tobesent, all_stimuli, stopevent):
+    try:
+        i = 0
+        if verbose:
+            print("Begin emission:")
+        for p in tobesent:
+            pks.send(p)
+            i += 1
+            time.sleep(inter)
+        if verbose:
+            print("Finished to send %i packets." % i)
+    except SystemExit:
+        pass
+    except KeyboardInterrupt:
+        pass
+    except:
+        log_runtime.exception("--- Error sending packets")
```

Comment:
This will be nicer with:
```
msg = "--- Error sending packets"
log_runtime.exception(msg)
log_runtime.info(msg)
```

Reply:
Totally. I've used a single call, to `.info()` with `exc_info` instead.